### DISCUSSION TITLE
Can i install Windows 10 intel within UTM, on my Macbook M1 ?

### DISCUSSION THREAD

**Discussion by vanhaze2000** [utmapp/UTM#4510] [UTM Help (macOS)]
Hi,

I am on a Macbook M1.
I understood that Windows 10 ARM is not available anymore for download so is it possible
to run Windows 10 **intel** within UTM, on my Macbook M1 ?
If so, is there any good guideline how to accomplish this ?

Kind Regards.

**Comment by christophenouvel** [comment]
Hi,

It is possible. Slow, but decent speed for office workloads. No game.

In my case I did a V2V from VirtualBox, WITHOUT uninstalling VBoxGuest Additions and WITHOUT installing spice guest tools... So it was a bit of pain before running this VM, but hey : now, it works! No crash at that time. 7 CPU cores given to the VM.
Better : audio works! It never worked on VirtualBox before (MacBook pro Intel).

UTM 4.0x and 4.1.0.

  Regards,

    Christophe.
